tests/integration_cases/test_raft.rs

```diff
@@ -3753,12 +3767,18 @@ fn test_remove_learner() {
     setup_for_test();
     let mut n1 = new_test_learner_raft(1, vec![1], vec![2], 10, 1, new_storage());
     n1.remove_node(2);
-    assert_eq!(n1.prs().nodes(), vec![1]);
-    assert_eq!(n1.prs().learner_nodes(), vec![]);
+    assert_eq!(
+        n1.prs().voter_ids().iter().cloned().collect::<Vec<_>>(),
```

Comment:
Ditto.

Reply:
It's the wrong data type. To make you happy I will unwrap the value and check it directly.